CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+
+# [v1.3.0](Unreleased) (2020-04-15)
```

Comment:
date is given to the released version. No need to add the date here

Reply:
Done